package.json

```diff
@@ -99,9 +99,7 @@
     "@rspack/binding-linux-x64-gnu": "1.3.10"
   },
   "peerDependencies": {
-    "@babel/plugin-transform-class-properties": "^7",
-    "react": "*",
```

Comment:
Above this is listed
"react": "^18.2.0",
"react-dom": "^18.2.0",

Side-note 18.3.1 seems to be the newest 18 version. (a year ago)
https://www.npmjs.com/package/react?activeTab=versions

I don't know if there is anything specific in 18.2.0 compared to 18.0.0 that we used?

So 18.0.0 might be supported so perhaps, the correct thing would be to
use ^18.3.1  under devDependencies

And ^18.0.0 in peerDependencies

Reply:
Which I think means we are using the newest 18.x (currently ^18.3.1) when developing and testing our module, but 18.0.0 should work.

When react is only under devDependencies we are saying that you don't need react at runtime, which is untrue.

The alternative would be to make react a direct dependency...

Reply:
Thank you, @ComLock ! Good tips !